src/containers/user/UserCardsContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import { UserCardProps } from '../../components/composite/common/cards';
+import { useUserCardsContainerQuery } from '../../hooks/generated/graphql';
+import { COUNTRIES_BY_CODE } from '../../models/constants';
+import { ContainerProps } from '../../models/container';
+import { buildUserProfileUrl } from '../../utils/urlBuilders';
+
+interface UserCardsContainerEntities {
+  users: UserCardProps[];
+}
+
+interface UserCardsContainerState {
+  loading: boolean;
+}
+
+interface UserCardsContainerProps extends ContainerProps<UserCardsContainerEntities, {}, UserCardsContainerState> {
+  userIDs: string[];
+}
+
+export const UserCardsContainer: FC<UserCardsContainerProps> = ({ children, userIDs }) => {
+  const { data, loading } = useUserCardsContainerQuery({ variables: { ids: userIDs } });
+
+  const users =
+    data?.usersById.map(
+      u =>
+        ({
+          tags: u.profile?.tagsets?.flatMap(x => x.tags),
+          displayName: u.displayName,
+          avatarSrc: u.profile?.avatar,
+          url: buildUserProfileUrl(u.nameID),
+          city: u.city,
+          country: COUNTRIES_BY_CODE[u.country],
+        } as UserCardProps)
+    ) || [];
```

Comment:
the role name is not provided. Is this intentional?

Reply:
Yes, the role needs context.  The search doesn't have context. Later is set to Member just to fill the empty space on the card.

Reply:
@Comoque1

Reply:
From what I saw on PROD and DEV, only admin roles are displayed on search cards.
![image](https://user-images.githubusercontent.com/23620658/141086665-1a92a65f-7a05-4838-b0f1-03afcaa32286.png)

Reply:
@Comoque1 this is different.
@hero101 I'm wrong. Actually we have context because this card is shown under the community.